docs/schema/identifiers.md

```diff
@@ -87,6 +87,8 @@ The ocid prefix itself is made up of two parts: a prefix agency identifier (curr
 
 The ocid is case-sensitive; in other words, the letter case of an ocid must be consistent.
 
+It is good practice to separate the ocds prefix and the internal identifier with an hyphen (`-`).
```

Comment:
```suggestion
It is encouraged to separate the ocds prefix and the internal identifier with a hyphen (`-`).
```

Reply:
Sorry, I had forgotten the documentation.